source/logs.txt

```diff
@@ -0,0 +1,49 @@
+.. _intellij-logs:
+
+======================
+Retrieve IntelliJ Logs
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This page describes how to retrieve log files from IntelliJ IDEA.
+
+About This Task 
+---------------
+
+Log files help the MongoDB development team diagnose bugs in the 
+{+intellij-short+}. When you retrieve log files, IntelliJ generates a zip with 
+the following contents:
+
+- General information about the IDE installation such as version, host OS, host 
```

Comment:
[optional suggestion]

You might consider bulleting out these aspects in a nested list.

Reply:
Sounds good! Updating.